Ryujinx.Graphics.Shader/CodeGen/Spirv/CodeGenContext.cs

```diff
@@ -234,7 +234,7 @@ public Instruction Get(AggregateType type, IAstNode node)
                     IrOperandType.Constant => GetConstant(type, operand),
                     IrOperandType.ConstantBuffer => GetConstantBuffer(type, operand),
                     IrOperandType.LocalVariable => GetLocal(type, operand),
-                    IrOperandType.Undefined => Undef(GetType(type)),
+                    IrOperandType.Undefined => Constant(GetType(type), operand.Value),
```

Comment:
```suggestion
                    IrOperandType.Undefined => Constant(GetType(type), 0),
```
`Undef` does not really carry any value, so `Value` will always be 0 anyway.